src/chapters/orm-relationships/many-to-one.rst

```diff
@@ -0,0 +1,205 @@
+
+Our many-to-one relationship is established in the ``Event`` class; each event will have one category, and there may be many events in a given category. 
+
+Currently, similar functionality is enabled via the ``EventType`` field of ``Event``. However, ``EventType`` is an enum, which means that new values can not be added without changing the code and re-compiling. Using the persistent ``EventCategory`` class to organize events will be a much more flexible and user-friendly approach. 
+
+Replacing ``EventType`` With ``EventCategory``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In the ``Event`` class, replace the ``type`` field with a new private field of type ``EventCategory``:
+
+.. sourcecode:: java
+   :lineno-start: 25
+
+   private EventCategory eventCategory;
+
+This will introduce compiler errors wherever ``type`` is referenced. We will fix these shortly.
+
+First, however, update ``Event`` so that:
+
+#. Its main constructor (i.e. NOT the no-arg constructor) takes a parameter of type ``EventCategory`` and uses it to initialize ``eventCategory``.
+#. It has a getter/setter pair for ``eventCategory``.
+#. The ``eventCategory`` field has the ``@NotNull`` validation annotation, along with an appropriate message.
+
+This establishes a relationship between the two classes, but that relationship is not yet *persistent*. To configure Hibernate to keep track of this relationship in the database, we add the ``@ManyToOne`` annotation to ``eventCategory``. 
+
+.. sourcecode:: java
+   :lineno-start: 27
+
+   @ManyToOne
+   @NotNull(message = "Category is required")
+   private EventCategory eventCategory;
+
+This annotation informs Hibernate that there can be many events for each category, but only one category per event. 
+
+Before we can start up the app and test our changes, we need to remove the code that references the deleted ``type`` field. A quick way to find these references is to find ``EventType`` in the *Project* pane of IntelliJ, right-click, and select *Find Usages*. This will open a pane listing each location ``EventType`` occurs in our project, whether that is an object of that type or a direct reference to the class itself. 
+
+.. figure:: figures/find-usages.png
+   :alt: The search results pane after running Find Usages on EventType
+   :width: 600px
+
+   All remaining references to ``EventType`` in our project
+
+The first occurrence is in ``EventController.displayCreateEventForm``:
+
+.. sourcecode:: java
+   :lineno-start: 35
+
+   model.addAttribute("types", EventType.values());
+
+This line passes a collection of all of the values of ``EventType`` into the view, to be rendered in the form used to create new events.
+
+Since we are now using ``EventCategory`` to group events, our code should instead be passing in all of the category objects in our app. To fetch category objects, we need an instance of ``EventCategoryRepository`` in our controller. Add an autowired instance to the top of the controller:
```

Comment:
Should "autowired" be capitalized and receive syntax highlighting?

Reply:
I'm using this as a verb, rather than referencing the specific annotation. The alternative would be `@Autowire`-ed, which seems a bit awkward. Thoughts?

Reply:
I think, "Add an ``@Autowired`` instance to..." sounds fine, since the annotation name is a noun rather than a verb.